indico/modules/events/editing/controllers/backend/editable_list.py

```diff
@@ -234,7 +235,7 @@ def _query_editables(self, has_files, extensions):
         revision_query = revision_query.subquery()
         return (Editable.query
                 .join(revision_query, revision_query.c.editable_id == Editable.id)
-                .options(joinedload('contribution'))
+                .options(joinedload('contribution').joinedload('person_links'))
```

Comment:
selectinload might be better here to avoid repeating all the contribution and editable data for each person

Reply:
I tested both and joinedload generated less queries and was faster... unless it's a memory thing, i haven't checked that

Reply:
(small difference tho, like 9 vs 5 queries for 1000+ contribs)

Reply:
selectinload uses an extra query for every n (i guess the max amount of values that fit in an `IN` clause) rows from the outer query, instead of getting everything in the same query. 

so it's slightly more queries but especially with many persons on each contribution I'd have expected it to be faster...

Reply:
ok you're right, i was checking the editable list instead of the get next editable.
0.24ms (selectinload) vs 0.31ms (joinedload)

in the other list it's the other way around, but it makes sense since there we're querying the contrib directly. thanks! (and don't forget to enjoy your vacation 😄)